scripts/fetch_synthesis_definition.sh

```diff
@@ -16,9 +17,7 @@ for definition in ${DEFINITION_NAMES}; do
       repo_url=${definition_version%%#*}
       repo_commit=${definition#*#}
       cd ${tmp_dir}; \
-        git remote add forked ${repo_url}; \
-        git fetch forked;  \
-        git switch ${repo_commit} -d;
+        git -c advice.detachedHead=false checkout ${repo_commit};
```

Comment:
why on the same line?

Reply:
Just to remove the detached head advice only for this statement

Reply:
I guess the question of @paologallinaharbur  is related to the `; \` and the one liner scrip, I don't think is necessary here if i'm not mistaken .

Reply:
yes exactly!

Reply:
Done!